xonsh/main.py

```diff
@@ -170,8 +170,8 @@ def parser():
                    choices=('readline', 'prompt_toolkit', 'best', 'random'),
                    default=None)
     p.add_argument('--timings',
-                   help='Prints timing infomation before the prompt is shown. '
-                        'This is usefull to track down perfomance issues '
+                   help='Prints timing information before the prompt is shown. '
+                        'This is usefull to track down performance issues '
```

Comment:
Why oh why did my spell checker not catch that?! I will resolve this post haste!